packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -522,4 +522,11 @@ export class GroupedVerticalBarChartBase extends React.Component<
   private _noLegendHighlighted = () => {
     return this.state.selectedLegend === '' && this.state.activeLegend === '';
   };
+
+  private _getAriaLabel = (point: IGVBarChartSeriesPoint, xAxisPoint: string): string => {
+    const xValue = point.xAxisCalloutData || xAxisPoint;
+    const legend = point.legend;
+    const yValue = point.yAxisCalloutData || point.data;
+    return point.callOutAccessibilityData?.ariaLabel || `${xValue}. ${legend}, ${yValue}.`;
```

Comment:
there is a property called stackCalloutAccessibilityData. Are you not using that.

Reply:
Unlike VerticalStackedBarChart, here the focus moves only on individual bars and not on stacks (groups of bars)